bezel-cfgs/ladybug.cfg

```diff
@@ -0,0 +1,12 @@
+# 1941 Counter Attack Overlay for Retroarch with 4:3 Display
```

Comment:
This overlay is not for 1941, isn't it? :)

Reply:
You're right, I forgot to change that. But I'll make a minor update to the artwork anyway, so stay tuned.

Reply:
Done, see second commit.